crates/ethereum-forks/src/hardfork/optimism.rs

```diff
@@ -199,6 +199,7 @@ impl OptimismHardfork {
         ChainHardforks::new(vec![
             (EthereumHardfork::Frontier.boxed(), ForkCondition::Block(0)),
             (EthereumHardfork::Homestead.boxed(), ForkCondition::Block(0)),
+            (EthereumHardfork::Dao.boxed(), ForkCondition::Block(0)),
```

Comment:
there was no dao on optimism so i don't think so. this fork was just an irregular state transition

Reply:
Not really, will take a look, the only reason why I thought it needed to be present was because in the original `From<Genesis> for ChainSpec` implementation, the `Dao` fork was included in the list of block based hardforks, for both Ethereum and Optimism.

Will have a look around to see if I can find something

Reply:
https://github.com/ethereum-optimism/op-geth/blob/optimism/params/superchain.go#L55

Will revert!